packages/dapp/src/helpers/parameterizer.ts

```diff
@@ -48,8 +48,12 @@ export async function initializeProposalsSubscriptions(dispatch: Dispatch<any>):
     const propState = await parameterizer.getPropState(propID);
     const challengeID = await parameterizer.getChallengeID(propID);
     const applicationExpiry = await parameterizer.getPropApplicationExpiry(propID);
-    const challengeCommitExpiry = await parameterizer.getPropChallengeCommitExpiry(propID);
-    const challengeRevealExpiry = await parameterizer.getPropChallengeRevealExpiry(propID);
+    const challengeCommitExpiry = !challengeID.isZero()
+      ? await parameterizer.getPropChallengeCommitExpiry(propID)
+      : null;
```

Comment:
i think you're supposed to use `undefined` instead of `null` pretty much always in typescript land, not sure it really matters though